sdk/xmake.lua

```diff
@@ -361,7 +361,11 @@ rule("firmware")
 			local interruptConfiguration = "CHERIOT_INTERRUPT_CONFIGURATION="
 			for _, interrupt in ipairs(board.interrupts) do
 				interruptNames = interruptNames .. interrupt.name .. "=" .. math.floor(interrupt.number) .. ", "
-				interruptConfiguration = interruptConfiguration .. "{" .. math.floor(interrupt.number) .. "," .. math.floor(interrupt.priority) .. "},"
+				interruptConfiguration = interruptConfiguration .. "{"
+					.. math.floor(interrupt.number) .. ","
+					.. math.floor(interrupt.priority) .. ","
+					.. (interrupt.edge_triggered and "true" or "false")
+					.. "},"
```

Comment:
While you're in the neighborhood, could you add a comment that these are initializers for `sched::InterruptController::Interrupt` structures?